CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+3.4.2
```

Comment:
I think we can update the version number to 3.5.0 for new features.

Reply:
updated